src/components/DefinitionEntry.js

```diff
@@ -201,13 +223,12 @@ class DefinitionEntry extends React.Component {
     // TODO: find a way of calling this method less frequently. It's relatively expensive.
     const definition = this.foldFacets(rawDefinition, this.props.activeFacets)
     const { licensed } = definition
-    const { readOnly, onRevert } = this.props
     return (
       <div className="row row-panel-details">
         <div className="col-md-6 d-flex justify-content-start align-items-center">
           <span className="panel-details__title">{this.renderLabel('Declared')}:</span>
           <div className="panel-details__value">
-            <p>{this.getValue('licensed.declared')}</p>
+            {this.renderFieldWithToolTipIfDifferent('licensed.declared')}
             {/* {this.renderWithToolTipIfDifferent(
               'licensed.declared',
```

Comment:
Line 231 seems to be a duplicate of lines 232 and 233. Is there any reason why we still keep the commented out code? If not, could you remove it in this PR to avoid the duplication?

Reply:
The commented out code intended to support editing each individual field (e.g "Declared"), including rendering edit button beside each field and either inline or a pop up upon clicking edit field button.  This PR only fixes the QuickEditModel pop up dialogue and allowing editing multiple fields in that dialogue.  So the commented out code is left as a reference in case editing each field is to be supported in the future.